draft-06/yang/ietf-netconf-private-candidate@2024-09-12.yang

```diff
@@ -0,0 +1,128 @@
+module ietf-netconf-private-candidate {
+  yang-version 1.1;
+  namespace "urn:ietf:params:xml:ns:yang:ietf-netconf-private-candidate";
+  prefix pc;
```

Comment:
Unfortunately there isn't much guidance around prefixes but will generally suggest something short, unique (to the IETF model-set) and descriptive - possible alternate suggestion (also don't feel strongly about it either way) - `ncpc`